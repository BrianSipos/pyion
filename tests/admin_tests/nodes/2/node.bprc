# ==================================================================================================== node.bprc ====================================================================================================
# Auto-generated @ Tuesday, 12. February 2019 09:30AM
# ===================================================================================================================================================================================================================

# Initialize
1

# Define a naming scheme for BP and its forwarder
a scheme ipn 'ipnfw' 'ipnadminep'

# New endpoint ipn.2.0
a endpoint ipn:2.0 x

# New endpoint ipn.2.1
a endpoint ipn:2.1 x

# New endpoint ipn.2.2
a endpoint ipn:2.2 x

# New endpoint ipn.2.64
a endpoint ipn:2.64 x

# New endpoint ipn.2.65
a endpoint ipn:2.65 x

# New endpoint ipn.2.100
a endpoint ipn:2.100 x

# New endpoint ipn.2.101
a endpoint ipn:2.101 x

# New endpoint ipn.2.102
a endpoint ipn:2.102 x

# Define the udp protocol: protocol name, payload [bytes], overhead [bytes], nominal_rate [bytes/sec]
a protocol udp 1400 100 125000

# Define an induct: protocol name, duct name, CLI command
a induct udp 196.128.0.102:4602 udpcli

# Define an outduct: protocol name, duct name, CLO command
a outduct udp 196.128.0.101:4601 'udpclo 0'

# Define all plans in ``node.ipnrc``
r 'ipnadmin node.ipnrc'

# Activate/Deactivate watch characters
w 1

# Start BP operation
s

# ======================================================================================================= EOF =======================================================================================================