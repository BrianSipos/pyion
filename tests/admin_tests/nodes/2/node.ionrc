# ==================================================================================================== node.ionrc ====================================================================================================
# Auto-generated @ Tuesday, 12. February 2019 09:30AM
# ====================================================================================================================================================================================================================

# 1=start node, 2=node nbr, 3=config file
1 2 node.ionconfig

# Start node
s

# Define horizon for congestion forecast to infinity
m horizon +0

# ======================================================================================================= EOF =======================================================================================================