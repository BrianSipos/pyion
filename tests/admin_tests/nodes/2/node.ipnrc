# ==================================================================================================== node.ipnrc ====================================================================================================
# Auto-generated @ Tuesday, 12. February 2019 09:30AM
# ====================================================================================================================================================================================================================

# Define a plan (possibility to tx) to node node1 using outduct udp
a plan 1 udp/196.128.0.101:4601

# ======================================================================================================= EOF =======================================================================================================