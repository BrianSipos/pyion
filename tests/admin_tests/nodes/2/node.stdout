[i] admin pgm using SDR parm overrides from node.ionconfig.
wmKey:           65281
wmSize:          50000000
wmAddress:       0
sdrName:        'ion'
sdrWmSize:       0
configFlags:     1
heapWords:       7812500
heapKey:         -1
logSize:         0
logKey:          -1
pathName:       '/tmp'
Stopping ionadmin.
Now checking ion
: Stopping ionadmin.
ION system is started
Stopping ionadmin.
Now checking bp
Stopping ionsecadmin.
: Stopping ipnadmin.
Stopping bpadmin.
BP agent is started
Stopping bpadmin.

ION system started script completed.
yzyzyzyzyzyzyzyzyzyzyzyzyzyzyzyzyzyzyzyzyzyz